pandas/_typing.py

```diff
@@ -83,7 +83,7 @@
 PythonScalar = Union[str, int, float, bool]
 DatetimeLikeScalar = Union["Period", "Timestamp", "Timedelta"]
 PandasScalar = Union["Period", "Timestamp", "Timedelta", "Interval"]
-Scalar = Union[PythonScalar, PandasScalar]
+Scalar = Union[PythonScalar, PandasScalar, np.datetime64, np.timedelta64]
```

Comment:
Before this PR, `Interval` resolved to `Any`. Some functions that are supposed to return a `Scalar` can return `np.datetime64` / `np.timedelta64` which was previously unnoticed.